src/config/options.js

```diff
@@ -48,29 +44,17 @@ export const options = {
       ARRAY_EXPRESS: 'ArrayExpress'
     }
   },
-  // setting for the request data form
-  requestDataForm: {
-    radioPediatricCancer: [
-      { label: 'Yes', value: 'Yes' },
-      { label: 'No', value: 'No' }
-    ],
-    radioPrimaryApproach: [
-      { label: 'Bench Research', value: 'Bench Research' },
-      { label: 'Computational Research', value: 'Computational Research' },
-      { label: 'Clinical Research', value: 'Clinical Research' },
-      { label: 'AI/ML Research', value: 'AI/ML Research' }
-    ]
-  },
```

Comment:
I've cleaned up these definitions for the request data forms as both the labels and values are identical. They are no defined within the components and passed directly to Grommet components.